packages/stakers/src/set/ensureStakerPkgsNetworkConfig.ts

```diff
@@ -0,0 +1,194 @@
+import {
+  dockerNetworkConnectNotThrow,
+  dockerNetworkDisconnect,
+  listPackageNoThrow,
+} from "@dappnode/dockerapi";
+import { params } from "@dappnode/params";
+import { logs } from "@dappnode/logger";
+import { ComposeFileEditor } from "@dappnode/dockercompose";
+import { networks, PackageContainer } from "@dappnode/types";
+import { getStakerConfigByNetwork } from "../getStakerConfigByNetwork.js";
+import { getStakerDnpNamesByNetwork } from "../get/getStakerDnpNamesByNetwork.js";
+
+/**
+ * Ensure the following staker configuration:
+ * - Add the staker network and its fullndoe alias to the docker-compose file to the execution client
+ * - Connect the execution and consensus pkgs to the staker network
+ *
+ * For the following staker pkgs from every network:
+ * - **selected** execution client
+ * - **selected** consensus client
+ */
+export async function ensureStakerPkgsNetworkConfig(): Promise<void> {
+  for (const network of networks) {
+    const stakerConfig = getStakerConfigByNetwork(network);
+    const { executionClients, consensusClients } =
+      getStakerDnpNamesByNetwork(network);
+
+    for (const { dnpName, clientsToCheck, alias } of [
+      {
+        dnpName: stakerConfig.executionClient,
+        clientsToCheck: executionClients,
+        alias: getStakerFullnodeAlias(network, stakerConfig.executionClient),
+      },
+      {
+        dnpName: stakerConfig.consensusClient,
+        clientsToCheck: consensusClients,
+        alias: null,
+      },
+    ]) {
+      // skip if no client selected
+      if (!dnpName) continue;
+
+      const pkg = await listPackageNoThrow({
+        dnpName: dnpName,
+      });
+      // skip if pkg not installed
+      if (!pkg) continue;
+
+      // Disconnect and remove staker network from other clients
+      await disconnectAndRemoveStakerNetworkFromCompose(clientsToCheck);
```

Comment:
why is there no error handling here or in `disconnectAndRemoveStakerNetworkFromCompose()` itself? calling a container to disconnect can return "server error" or "not found" error: https://docs.docker.com/engine/api/v1.37/#tag/Network/operation/NetworkDisconnect

Reply:
we do not want to move forward if the docker compose config has not bee removed successfully, otherwise it may lead to more than exec/cons client from the same network in the staker_network